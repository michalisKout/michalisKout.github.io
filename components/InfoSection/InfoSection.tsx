import InfoContent from "./components/InfoContent";

const InfoSection = () => (
  <section className="info--container">
    <div className="info--wrapper">
      <div className="lg:w-1/2">
        <div
          className="info--image"
          style={{
            backgroundImage:
              "url('https://images.unsplash.com/photo-1593642532400-2682810df593?ixlib=rb-1.2.1&ixid=MXwxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=750&q=80')",
          }}
        />
      </div>
      <InfoContent />
    </div>
  </section>
);

export default InfoSection;
