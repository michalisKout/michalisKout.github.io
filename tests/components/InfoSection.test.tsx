import { render } from "@testing-library/react";
import InfoSection from "../../components/InfoSection/InfoSection";

it("should render InfoSection", () => {
  const { asFragment } = render(<InfoSection />);
  expect(asFragment()).toMatchSnapshot();
});
