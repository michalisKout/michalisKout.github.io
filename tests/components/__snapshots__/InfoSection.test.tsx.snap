// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render InfoSection 1`] = `
<DocumentFragment>
  <section
    class="info--container"
  >
    <div
      class="info--wrapper"
    >
      <div
        class="lg:w-1/2"
      >
        <div
          class="info--image"
          style="background-image: url(https://images.unsplash.com/photo-1593642532400-2682810df593?ixlib=rb-1.2.1&ixid=MXwxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=750&q=80);"
        />
      </div>
      <div
        class="info--content--wrapper"
      >
        <h2
          class="info--title"
        >
          Build Your New
          <span
            class="info--tag"
          >
            Idea
          </span>
        </h2>
        <p
          class="info--content"
        >
          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quidem modi reprehenderit vitae exercitationem aliquid dolores ullam temporibus enim expedita aperiam mollitia iure consectetur dicta tenetur, porro consequuntur saepe accusantium consequatur.
        </p>
        <div
          class="mt-8"
        >
          <a
            class="info--button"
            href="#"
          >
            Start Now
          </a>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
